imports/plugins/core/ui-navbar/client/components/navbar/navbar.js

```diff
@@ -90,27 +92,36 @@ Template.CoreNavigationBar.helpers({
     };
   },
 
-  tagNavProps() {
+  tagNav() {
```

Comment:
I saw these two patterns in the code. I thought any can be used. I'll update to this method.